plugin/ocgrpc/grpc_test.go

```diff
@@ -72,12 +72,12 @@ func TestNewClientStatsHandler(t *testing.T) {
 }
 
 func TestNewServerStatsHandler(t *testing.T) {
-	ctx := context.Background()
-
-	handler := NewServerStatsHandler()
-
-	te := &traceExporter{}
-	trace.RegisterExporter(te)
+	var (
```

Comment:
Can you revert these changes?

In Go, this style is discouraged. var grouping is mostly discouraged unless the variables need to be strictly grouped together for readability. var is entirely discouraged unless it is for the zero value.